### DISCUSSION TITLE
Static translations: editable in admin

### DISCUSSION THREAD

**Discussion by bertheyman** [craftcms/cms#8822] [Ideas]
Really happy with the way multilingual sites work in Craft.
Static translations sometimes feel somewhat limited when working with sites that have lots of them.

Current options:
1. Working with globals: not ideal when there are lots of translations.
2. Using the static file translations: more efficiënt for larger amounts of translations, but the client loses the ability to edit these.
3. Having the static file translations in the admin: currently only possible with plugins like https://plugins.craftcms.com/translations-admin.

Would be really great to have option 3 as part of the core functionality.

**Comment by brandonkelly** [comment]
I’ll leave this open because it’s certainly a possibility, but I’d suggest a 4th option – use a service like [Crowdin](https://crowdin.com/). That’s what we use for Craft’s [control panel translations](https://crowdin.com/project/craft-cms). It provides a much more efficient way to collaborate on translations than Craft itself could ever provide, and they also integrate with translation services if you want to outsource it.

**Comment by brandonkelly** [comment]
Just wrote a guide on how to setup Crowdin for managing static translations: https://craftcms.com/knowledge-base/managing-static-message-translations-with-crowdin

**Comment by piotrpog** [comment]
In most situations, Crowdin is overkill. With sites that have one language, client just needs interface in the control panel to be able to edit few static translation strings. That's why but enupal translate plugin for most of our pages. 

I really wish that was built in interface for that, so no plugins are needed. I feel its kinda basic functionality.

**Comment by brandonkelly** [comment]
Sure, as I said I’m not against the idea. But the context here is sites with lots of translations, and Crowdin is likely not overkill at that point.

**Comment by bertheyman** [comment]
Hi @brandonkelly, thanks for the advice!
Crowdin sounds good for some specific use cases, but the cost might be an issue for smaller or medium sized projects.

When building sites for multilingual countries, even smaller sites will easily support two er three languages so for most of these occasions an in-admin editor would be the ideal solution. (e.g. in the Belgian market this is quite common)
However I can see this is a really hard choice to make: lots of different use cases and slightly different needs, so I'd totally understand if this isn't ideal for Craft.

**Comment by piotrpog** [comment]
I was wondering, is that still on the table? Are there any concrete plans to implement it? I would really wish that we did not had to install external plugins for such a basic functionality.

**Reply by brandonkelly** [reply]
We might consider it down the road, but there aren’t any concrete plans right now.
